### PR TITLE
Barchart implementation

### PR DESCRIPTION
**Pull request by nsdere**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--bitdefender--hlxsites.hlx.page/en-au/solutions/
- After: [https://barchart--bitdefender--hlxsites.hlx.page/en-au/solutions/](https://barchart--bitdefender--hlxsites.hlx.page/en-au/solutions/)

### CODE REVIEW

**Review by basecode — CHANGES_REQUESTED** [review]
Great first PR! 👏 

I added a few small comments. Please also pay attention to PSI report. There is a drop of accessibility.

File: blocks/cards/cards.js [hunk 1]
```diff
@@ -15,4 +15,36 @@ export default function decorate(block) {
...
   block.append(ul);
+
+  const containers = document.querySelectorAll('.cards.barchart.block .cards-card-body');
+  containers.forEach((container) => {
+    const progressBars = container.querySelectorAll('li > ul');
+    progressBars.forEach((progressBar, index) => {
+      const value = parseFloat(progressBar.textContent);
+      const label = progressBar.parentElement.firstChild.textContent.trim();
+
+      let bar = progressBar.querySelector('.bar');
```
Comment (vtsaplin) [inline blocks/cards/cards.js:26]: What is the purpose of this check?

File: blocks/cards/cards.js [hunk 2]
```diff
@@ -15,4 +15,36 @@ export default function decorate(block) {
...
+        labelElement.textContent = label;
+        bar.appendChild(labelElement);
+
+        const valueElement = document.createElement('span');
+        valueElement.classList.add('value-label');
+        valueElement.textContent = value;
+        bar.appendChild(valueElement);
+
+        if (index === 0) {
+          bar.classList.add('first-bar');
```
Comment (vtsaplin) [inline blocks/cards/cards.js:45]: This can be easily addressed by using `first-child` pseudo-class like so:

```css
.bar::first-child {
}
```

**Review by vtsaplin — CHANGES_REQUESTED** [review]
Please move the implementation to the dedicated `barchart` block.

**Review by basecode — APPROVED** [review]

**Review by vtsaplin — APPROVED** [review]

**Pull request merged**